minigalaxy/ui/gametile.py

```diff
@@ -272,10 +329,18 @@ def __get_install_dir(self):
 
     def reload_state(self):
         self.game.install_dir = self.__get_install_dir()
-        dont_act_in_states = [self.state.QUEUED, self.state.DOWNLOADING, self.state.INSTALLING, self.state.UNINSTALLING]
+        dont_act_in_states = [self.state.QUEUED, self.state.DOWNLOADING, self.state.INSTALLING, self.state.UNINSTALLING,
+                              self.state.UPDATING, self.state.DOWNLOADING]
         if self.current_state in dont_act_in_states:
             return
-        if self.game.install_dir and os.path.exists(self.game.install_dir):
+        if self.game.installed_version:
+            installer = self.api.get_info(self.game)["downloads"]["installers"]
```

Comment:
This line and the line below it have to be executed in a separate thread. It is currently making my Minigalaxy freeze up for a couple of seconds, because I have over 90 games installed.